post.go

```diff
@@ -40,11 +41,16 @@ func (p Post) GenerateInsightParams() BatchParams {
 	return NewBatchParams(fmt.Sprintf("%s/insights/post_video_views_unique,post_engaged_users,post_video_complete_views_organic,post_video_complete_views_paid,post_consumptions,post_impressions,post_impressions_paid,post_impressions_unique,post_impressions_paid_unique,post_video_views,post_video_views_paid,post_video_views_organic,post_video_view_time,post_video_avg_time_watched,post_stories_by_action_type?period=lifetime&limit=20", p.ID))
 }
 
-// GenerateParams comments pending
+// GenerateParams comment pending
 func (p *Post) GenerateParams() BatchParams {
 	return NewBatchParams(fmt.Sprintf("%s?fields=%s", p.ID, "object_id,message"))
 }
 
+// GeneratePostCreatedTimestampParams comment pending
+func (p *Post) GeneratePostCreatedTimestampParams() BatchParams {
```

Comment:
Do we need to do __another__ call to the post endpoint.. can't we just add `created_time` to the list of field in `GenerateParams()` then we'll also have the `created_time` in that payload..